pkg/kpipelines/syntax/pipeline_internal_test.go

```diff
@@ -54,59 +53,3 @@ func TestFindDuplicates(t *testing.T) {
 		})
 	}
 }
-
-func TestFindDuplicatesWithStages(t *testing.T) {
```

Comment:
This was doing the same thing as TestValidationFailures, so I just moved the files around and merged them into the other test.